sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_models.py

```diff
@@ -254,20 +254,16 @@ class JobStatusResult(object):  # pylint: disable=useless-object-inheritance, to
     :ivar int documents_not_yet_started_count: Number of documents that have not yet started being translated.
     :ivar int documents_cancelled_count: Number of documents that were cancelled for translation.
     :ivar int total_characters_charged: Total characters charged across all documents within the job.
-    :ivar bool has_completed: boolean to check whether a job has finished or not.
```

Comment:
The `JobStatusResult` is used when the user calls `list_submitted_jobs` right? I don't fully understand why the property is gone after we saw in the user studies that it was helpful to have it

Reply:
Let me check the UX study notes - I remember the opposite feedback (at least for Python)

Reply:
interesting. I remember  .NET people always using that property. 
I also remember they confuse it with the HasCompleted from the operation so I guess we could remove it and if people complain then added back.
I don't feel strongly about either option

Reply:
oh interesting, didn't realize you also had a `hasCompleted` on the operation. Yeah, I checked the notes on the understanding of `has_completed` and there were 4 no and two maybes. Guessing those are mostly python participants 😄